oneview/data_source_connection_templates.go

```diff
@@ -58,14 +60,15 @@ func dataSourceConnectionTemplates() *schema.Resource {
 			},
 			"name": {
 				Type:     schema.TypeString,
-				Required: true,
+				Optional: true,
```

Comment:
If name not given, we may get idempotency issues. Please verify and update as Optional and Computed.

Reply:
done